src/main.rs

```diff
@@ -22,6 +23,8 @@ Wagi provides a few ways of speeding up performance. The easiest way is to enabl
 cache, which will cause all modules to be preloaded and cached on startup.
 "#;
 
+const ENV_VAR_HELP: &str = "specifies an environment variable that should be used for every module WAGI runs. These will override any set by the module config. Multiple environment variables can be set per flag (e.g. -e FOO=bar BAR=baz) or the flag can be used multiple times (e.g. `-e FOO=bar -e BAR=baz`). Variables can be quoted (e.g. FOO=\"my bar\")";
```

Comment:
This only formatted properly if I didn't add in line breaks. If anyone knows how to get this to split multiline, that'd be great

Reply:
I tried this elsewhere in Wagi and also could not find a good way. So I gave up.